streams/re.m3u

```diff
@@ -0,0 +1,7 @@
+#EXTM3U
+#EXTINF:-1 tvg-id="ExoTV.re" status="online",Exo TV
+https://statslive2.infomaniak.ch/playlist/exofm/exofm-64.aac/playlist.m3u
+#EXTINF:-1 tvg-id="KanalAustralTV.re" status="online",Kanal Austral TV
+https://rr3---sn-vgqskn6s.googlevideo.com/videoplayback?expire=1681619669&ei=dCY7ZMPUOcLpigTijLuQDQ&ip=2001%3A1970%3A5823%3Abc00%3A565%3A8cc6%3A4f8b%3A8433&id=o-ALZA6m593_lmj0L9JPz6KsKtFiG4lTmd3vBMB_KoWqtY&itag=136&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278&source=youtube&requiressl=yes&spc=99c5CZRrToHySkJGC2zCLz7P_BnF_6dkzTfL6wX7c5o6twhbFGtVBvs&vprv=1&mime=video%2Fmp4&ns=YMRjbPO79IhrDc_OS0tDvbsM&gir=yes&clen=3929686452&otfp=1&dur=18944.966&lmt=1592703622874005&keepalive=yes&fexp=24007246&c=WEB&txp=7216222&n=zyzNU2z_I4szVg&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cspc%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cotfp%2Cdur%2Clmt&alr=yes&sig=AOq0QJ8wRQIgb3KVOpxbGCQ-THuWDRa520sb0fq3BvyJl9iixYhEyzECIQC6jzmHy9TaEqYLjYXCefGyWQaBF1zh2mB3ufm1HpSjkA%3D%3D&cpn=K0XMn0SQohJJ11Fu&cver=2.20230414.01.00&cm2rm=sn-jvbxjv-tihd7l,sn-tt1ls7e&ipbypass=yes&redirect_counter=2&cms_redirect=yes&cmsv=e&mh=ST&mm=34&mn=sn-vgqskn6s&ms=ltu&mt=1681597726&mv=m&mvi=3&pl=43&lsparams=ipbypass,mh,mm,mn,ms,mv,mvi,pl&lsig=AG3C_xAwRQIhAPVHrdkBqgoeytilsMLGcwGKoX8pv-neUrjuIt71J4i8AiADWaKZ5ruaORksrZTqE7GhCf3eAiq1kdZ6kQcS716B0w%3D%3D&range=14246357-16152576&rn=20&rbuf=50707&pot=MlsTxRibBKW6VBP9EUalhUEnG1dYZwimKgiFkTQt9jD-adZ7nIGv1yOipYN276MMiez-vtG5muiRgv34TGxwodA_7tB-hsf5fnuNAcpxII-tlhvroMm_J5uucY47
```

Comment:
Cannot be used as-is.

Reply:
Again, edit as you like: the ball is now rolling ;)